eventuate-tram-optimistic-locking/src/main/java/io/eventuate/tram/optimisticlocking/OptimisticLockingDecoratorConfiguration.java

```diff
@@ -0,0 +1,10 @@
+package io.eventuate.tram.optimisticlocking;
+
+import org.springframework.context.annotation.Bean;
+
+public class OptimisticLockingDecoratorConfiguration {
+  @Bean
+  public OptimisticLockingDecorator customerOptimisticLockingDecorator() {
```

Comment:
It doesn't make sense for it to be called customer...

Reply:
Missed it sorry